cmd/repository/default-pkg-registry-entry.go

```diff
@@ -0,0 +1,25 @@
+package repository
```

Comment:
Is this entry still in use? I couldn't find any reference any more in the review. If it is the case, we should remove it